data.js

```diff
@@ -0,0 +1,47 @@
+const artists = [
+  {
+    name: 'Thu Ya Kyaw',
+    position: 'Developer Advocate for Google Cloud (AI/ML)',
+    description:
```

Comment:
- [optional] please always follow the design to the fullest. we have 6 speakers in the [original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) but you have just 4. please always follow the design as provided.

Reply:
Fixed.